### DISCUSSION TITLE
Rofi for windows standalone

### DISCUSSION THREAD

**Discussion by Vortigern-The-Grey** [eythaann/Seelen-UI#101] [Ideas]
Would it be possible to release the rofi-like app launcher as a seperate, if small, download?

**Comment by eythaann** [comment]
Seelen UI is really lightweight on disk, and all the features are optional so you can disable all and leave the app launcher enabled to get it.

**Comment by wrecklass** [comment]
Probably easier to just install Flow-Launcher or PowerToys and use it's launcher. Both are pretty easy. PowerToys has a lot of other useful items. Overall I found SeelenUI far too buggy to use on my system. Crashes constantly.
